### ISSUE TITLE
sinr vs grid stuff

### ISSUE THREAD

**Issue by alexshepard** [inaturalist/iNaturalistMLWork#143]
evaluate SINR vs GRID approach to geo modeling across a number of different criteria:

- approach: SINR point-based approach vs the discretized GRID approach
- datasets: with the SINR dataset and with the 2.17 dataset
- inputs: with coordinates only, with coords + elevation
- metrics: prauc on 2.17 taxon range recalls & csvs, and cv task gain with 2.17 CV model

things further down the road I'd like to look at:
- framework: compare Eli's SINR implementation in PyTorch vs my SINR implementation in TF
- different inputs, including all bioclim coordinates, just an ocean mask, distance from coastline
- investigate the differences between different approaches for specific taxa. look beyond aggregate metrics and consider whether different approaches are better or worse for different subcategories of metrics: large or small range species, more or less training data, etc. can we plot this? can we map it?
- is an ensemble approach helpful? are both approaches learning stuff that the other isn't learning? or is one approach just better.
